internal/donor/donorpage/progress.go

```diff
@@ -89,6 +87,22 @@ func Progress(tmpl template.Template, lpaStoreResolvingService LpaStoreResolving
 			}
 		}
 
+		if !donor.Tasks.ConfirmYourIdentity.IsCompleted() &&
+			!donor.FailedVoucher.FailedAt.IsZero() &&
+			!donor.WantVoucher.IsNo() &&
+			donor.Voucher.FirstNames == "" {
```

Comment:
`donor.Voucher.FirstNames == ""` is here to show this banner if someone selects new voucher but doesn't give any details.